packages/kanso-app-settings/build/compile.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 function get_doc(ddoc_url, settings, callback) {
     couchr.get(ddoc_url, function(err, resp){
         if (err && err.error === 'unauthorized') return auth(ddoc_url, settings, callback);
-        if (err && err.error === 'not_found') return callback(null, {}); // no design doc. First push.
+        if (err && err.response.statusCode === 404) return callback(null, {}); // no design doc. First push.
```

Comment:
This is a bit dodgy. This is a 3rd party package for which we've committed the source: https://github.com/kanso/kanso-app-settings

It would be better to update and publish the package so it supports couchdb2 for other people as well. In practice it's a dead project so I understand if it's too much work...